src/PKSim.Infrastructure/Reporting/Summary/IReportBuilder.cs

```diff
@@ -21,6 +21,10 @@ public virtual ReportPart Report(T objectToReport)
       {
          var reportPart = new ReportPart();
          FillUpReport(objectToReport, reportPart);
+         //Only one item returned in the report. Then return it instead of wrapping it in yet another report part
+         if (reportPart.SubParts.Count == 1)
```

Comment:
The way this construct is working, we are creating a part, adding the actual report part and then returning the top part.
This makes sense for multiple levels but in our case. not really. 
So when we only have one level, we return the level instead of returning a bunch of empty wrappers